src/olympia/versions/models.py

```diff
@@ -81,6 +82,19 @@ def latest_public_compatible_with(self, application, appversions):
             files__status=amo.STATUS_APPROVED,
         ).order_by('-created')
 
+    def auto_approve(self):
```

Comment:
nit: doesn't block this, but I'm not a fan of the name here, it implies that calling this will auto approve versions in the queryset. And it should have a docstring.

Reply:
name suggestions?  `get_auto_approvable`?

Reply:
even `auto_approvable` would work for me I think. `Version.objects.auto_approvable()` seems good.

Reply:
https://github.com/mozilla/addons-server/pull/11527/commits/c4f09470196bdb3002b3181213a1f625710ad228